### DISCUSSION TITLE
Disable Irrelevant Warnings and Output

### DISCUSSION THREAD

**Discussion by cseitz** [vercel/next.js#33661] [Ideas]
### Describe the feature you'd like to request

```
API response for /file exceeds 4MB. This will cause the request to fail in a future version. 
https://nextjs.org/docs/messages/api-routes-body-size-limit
```

- I am not utilizing serverless functions.
- I do not plan on utilizing serverless functions.
- I do not plan on utilizing my project for more than a dozen users.
- I am utilizing an external resolver for said API route.

Thus, this response size limit does not apply to me.

Utilizing a CDN is overkill for my project. I do not want to serve my files with a CDN for such an application, especially since it incurs cost.

This warning is very helpful, but it is in no way applicable to my project or how I plan to use it. It's uselessly cluttering up my output.

Can I turn this warning off? Also, will Next.js eventually break my API __even when its run in an environment__ where the 4 MB size limit does not apply? That would be **very** annoying.

### Describe the solution you'd like

Additional configuration could be added to `next.config.js`. 

Any non-error messages should reference an `outputConfig` entry and allow some outputs to be turned off.

```ts
import getConfig from 'next/config';

if (getConfig().outputConfig['api-routes-body-size-limit'] != false)
    console.log('API response for /file exceeds 4MB...');
```

`next.config.mjs`
```ts
// Used to toggle various behaviors for Next.js's output.
export const outputConfig = {
    // false = "silence" / dont show
    "api-routes-body-size-limit": false,
    // names should correspond to links shown in the output so its easy to keep track of them.
}
```

Again, this warning is not applicable to me and I just want to be able to turn it off.

### Describe alternatives you've considered

- Scroll more than I need to because my output is obscured by multiple instances of said warning.
- Reinvent the wheel and write an encapsulating program to filter it out of the output (not worth the effort)

**Comment by ImTemporaryHere** [comment]
have same issue , waiting for their reply

**Comment by MichaelCasaDev** [comment]
I ended by creating an express.js web server for API handling on my project.
Currently, there is a body size limit of 4M because the instance works on a serverless environment and needs to minimalize the run time.

**Reply by cseitz** [reply]
Sure, I understand that; but I'm not using my project on a serverless runtime or platform.  I should be able to disable this output and not have to use an express server to get around just this warning message.
